data/client/king_phisher/plugins/data_collection.py

```diff
@@ -0,0 +1,158 @@
+import argparse
+import copy
+import datetime
+import hashlib
+import sys
+
+from king_phisher import color
```

Comment:
These import should all be consistent with each other as `import x.y as y` or `from x import y`, either one works but be consistent.